scripts/build-and-release-docs.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+/* eslint-env node */
+const sh = require('shelljs')
+const semver = require('semver')
+
+// this script is expected to run on release branch.
+const main = () => {
+    // get the latest version from npm registry
+    const {stdout: latestVersion} = sh.exec('npm view pwa-kit-react-sdk version', {silent: true})
```

Comment:
This should be after the `isPrerelease` check, since the value isn't used if that check fails.